internal/namespaces/lb/v1/custom_frontend.go

```diff
@@ -0,0 +1,18 @@
+package lb
+
+import "github.com/scaleway/scaleway-cli/internal/core"
+
+func frontendGetBuilder(c *core.Command) *core.Command {
+	c.View = &core.View{
+		Sections: []*core.ViewSection{
+			{
+				FieldName: "LB",
```

Comment:
I know Lb is included in the frontend response but do we want to print it ?

Reply:
#1286 offers the possibility to hide fields. We can open another PR if needed.